### ISSUE TITLE
Add support for "other discussions", "view discussions in x other communities"

### ISSUE THREAD

**Issue by gerion0** [QuantumBadger/RedReader#943]
On the website some posts are posted to multiple subreddit and linked via "other discussions" in the old style website and "view discussion in x other communities" in the normal website:
![image](https://user-images.githubusercontent.com/32870336/149680466-814b1d2d-3980-4931-9715-4b9ac75f3bc7.png)
![image](https://user-images.githubusercontent.com/32870336/149680502-3073875b-03c7-4f07-8fb6-7bbf7679f1b1.png)

[Here](https://old.reddit.com/r/pics/comments/s5bjq5/that_is_a_wonderful_accomplishment_and_much/) is an example of such a post.

I haven't found such a functionality in RedReader. Can you add support for it?

**Comment by woj-tek** [comment]
I was just going to suggest that. it's one of the more handy features of reddit that allows to get different discussion / perspective depending on the subreddit section.

**Comment by nascentt** [comment]
Pretty sure this is a duplicate of https://github.com/QuantumBadger/RedReader/issues/791

**Comment by gerion0** [comment]
Aren't crossposts more explicitely displayed? So it is directly clear, what the original post was (if I remember correctly they are kind of embedded in the site and it is mentioned "crosspost of ..." somewhere).

**Comment by QuantumBadger** [comment]
Agreed that this is not a duplicate, the crosspost issue is separate.
